charts/operator/templates/role.yaml

```diff
@@ -94,8 +94,6 @@ rules:
       - get
       - patch
       - update
-      - list
-      - watch
```

Comment:
This seems suspicious. Doesn't flux controller use list or watch capabilities from Kubernetes client-go library?

Reply:
suspicious? Interesting choice of words. Yeah ill review the file.

Reply:
This was not supposed to be here, only redpanda chart changes were supposed to here, removing.